### ISSUE TITLE
resume training feature

### ISSUE THREAD

**Issue by prankshtain** [Deci-AI/super-gradients#1024]
Hello, Is there any way that I can continue training a model from the last checkpoint to more epochs?

some of the time, google colab disconnects so we need to start training again with the last weights from the last trained epoch.

**Comment by NatanBagrov** [comment]
Hello @prankshtain ,
In case you stopped your training before it ended, you can just pass `resume=true` in the CLI to resume it.
If you want to train for additional epochs, then you need to specify `training_hyperparams.max_epochs=x+y` where `x` is the original number of epochs and `y` is the extra epochs you want to train.
Note, that you might tune the learning rate, especially if you are using `cosine` or `poly` scheduler.

**Comment by Louis-Dupont** [comment]
Hi @prankshtain, note that when you train on Google Colab, you lose any local data when the kernel is deleted. So in this case you will also lose your local checkpoints, so make sure to download them once you're done.
On the other hand, when colab simply restarts the runtime, you will still have your local data and will be able to resume, you can check this issue for more information https://github.com/Deci-AI/super-gradients/issues/940
Does that answer your question?

**Comment by prankshtain** [comment]
thanks very much
